package.json

```diff
@@ -42,7 +42,7 @@
     "compression": "1.7.4",
     "cors": "2.8.5",
     "deep-extend": "0.6.0",
-    "design-system": "git://github.com/Wikia/design-system.git#21.7.2",
+    "design-system": "git://github.com/Wikia/design-system.git#a1f7490",
```

Comment:
We should wait until https://github.com/Wikia/design-system/pull/366 is merged, then we can do a release and update this to point to that new release here.